CryptoPkg/Test/UnitTest/Library/BaseCryptLib/RsaPkcs7Tests.c

```diff
@@ -324,15 +337,28 @@ TestVerifyRsaCertPkcs1SignVerify (
   Status = X509VerifyCert (TestCert, sizeof (TestCert), TestCACert, sizeof (TestCACert));
   UT_ASSERT_TRUE (Status);
 
+  TestStatus = UNIT_TEST_PASSED;
+Exit:
   //
   // Release Resources.
   //
-  RsaFree (RsaPubKey);
-  RsaFree (RsaPrivKey);
-  FreePool (Signature);
-  FreePool (Subject);
+  if (RsaPrivKey != NULL) {

```

Comment:
Set these to NULL by default?

Reply:
Completed in PR commit https://github.com/Flickdm/mu_basecore/pull/1/commits/59d446c9d3451f0c212cbb994684129a40e3e8ac